### PR TITLE
Test setup changes related to issue 381, plus a doc change regarding shiftfs installation.

### PR DESCRIPTION
**Pull request by ctalledo**

### CODE REVIEW

File: tests/sysmgr/cmdline.bats [hunk 1]
```diff
@@ -28,12 +28,23 @@ function sysbox_mgr_stopped() {
    # Stop the sysbox-mgr
    sysbox_mgr_stop
 
+	# Verify the sysbox data root is gone
```
Comment (rodnymolina) [inline tests/sysmgr/cmdline.bats:31]: Check indentation -- also below.

File: docs/user-guide/install-package.md [hunk 2]
```diff
@@ -2,10 +2,13 @@
 
 This document describes how to install Sysbox using the [packaged versions](https://github.com/nestybox/sysbox/releases).
 
-If you are installing Sysbox on a Kubernetes cluster, use the
-[sysbox-deploy-k8s](install-k8s.md) daemonset instead.
+This is usually the case when installing Sysbox on hosts used for development or
```
Comment (rodnymolina) [inline docs/user-guide/install-package.md:5]: What about "production"? ;-)

**Review by rodnymolina — APPROVED** [review]
Have a few comments.

File: docs/user-guide/install-package.md (hunk 2 above)
Comment (ctalledo) [reply docs/user-guide/install-package.md]: Fixed, thanks.

File: tests/sysmgr/cmdline.bats (hunk 1 above)
Comment (ctalledo) [reply tests/sysmgr/cmdline.bats]: Fixed.

**Comment by ctalledo** [comment]
Travis CI is currently broken, but local CI passed. Will commit and deal with TravisCI soon.

**Pull request merged**